### PR TITLE
Feature/spectron test core

### PR DESCRIPTION
**Pull request by Jess-Jian**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4,26 +4,26 @@
 
 "@babel/code-frame@7.12.11":
   version "7.12.11"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
+  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.11.tgz"
```
Comment (dkdkwizard) [inline yarn.lock:7]: Do not commit yarn.lock if you didn't change package.json

File: src/web/app/views/dialogs/ConfirmPrompt.tsx [hunk 2]
```diff
@@ -77,6 +77,7 @@ function ConfirmPrompt({
         <div className="caption">{caption}</div>
         <pre className="message">{message}</pre>
         <input
+          id="power-caption"
```
Comment (dkdkwizard) [inline src/web/app/views/dialogs/ConfirmPrompt.tsx:80]: this is a general component, so id like `confirm-prompt-input` is better

File: src/web/app/views/dialogs/Prompt.tsx [hunk 3]
```diff
@@ -56,7 +56,7 @@ function Prompt({
 
   return (
     <Modal>
-      <div className={classNames('prompt-dialog-container', 'animate__animated', 'animate__bounceIn')}>
+      <div id="addparameter" className={classNames('prompt-dialog-container', 'animate__animated', 'animate__bounceIn')}>
```
Comment (dkdkwizard) [inline src/web/app/views/dialogs/Prompt.tsx:59]: this is a general component, so id like `prompt-container` is more proper

**Review by dkdkwizard — CHANGES_REQUESTED** [review]

File: src/web/app/views/dialogs/Prompt.tsx (hunk 3 above)
Comment (Jess-Jian) [reply src/web/app/views/dialogs/Prompt.tsx]: finished

File: src/web/app/views/dialogs/ConfirmPrompt.tsx (hunk 2 above)
Comment (Jess-Jian) [reply src/web/app/views/dialogs/ConfirmPrompt.tsx]: finished

File: yarn.lock (hunk 1 above)
Comment (Jess-Jian) [reply yarn.lock]: Finished

**Review by dkdkwizard — APPROVED** [review]

**Pull request merged**